samples/sample-jwt-generator/pom.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Shall we add the year here?

Reply:
Added with https://github.com/wso2/product-microgateway/pull/1613/commits/45bcd3462390fb8a500155b1accc2e40b6f75444